nifi-nar-bundles/nifi-websocket-bundle/nifi-websocket-processors/src/main/resources/docs/org.apache.nifi.processors.websocket.ConnectWebSocket/additionalDetails.html

```diff
@@ -35,7 +35,8 @@ <h2>Summary</h2>
 <p>
     You can define custom websocket headers in the incoming flowfile as additional attributes. The attribute key
     shall start with "header." and continue with they header key. For example: "header.Authorization". The attribute
-    value will be the corresponding header value.
+    value will be the corresponding header value. If a new flowfile is passed to the processor, the previous sessions will be closed,
```

Comment:
Can we please move this warning to the main description of the processor. It is an important aspect/limitation of the processor rather than a detail.